.github/workflows/schemacode_ci.yml

```diff
@@ -4,52 +4,82 @@ on:
   push:
     branches:
       - "master"
-    paths:
-      - "tools/schemacode/**"
-      - "src/schema/**"
+      - "autodata2"
```

Comment:
```suggestion
```

Reply:
ah, yes, I had said optionally in the comment and thought for some reason you wanted to keep it, but sure, dropping it makes sense.